client/state/test/initial-state.js

```diff
@@ -383,12 +383,12 @@ describe( 'initial-state', () => {
 							siteType: 'blog',
 							siteTitle: 'Logged out test title',
 						},
-						progress: [
-							{
+						progress: {
+							'logged-out-step': {
```

Comment:
It would be nice to avoid duplication here - the key is the same as stepName. Let's not delay this PR though - just keep not of that for the future.

Reply:
Totally! I'll look in to what it would take to do that soon after :)